lib/ansible/modules/network/qnos5/_qnos5_template.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python
```

Comment:
This appears to be a deprecated module. If all the functionality is in `qnos5_config` then you can delete this file.